payment_cycle/gql_mutations.py

```diff
@@ -0,0 +1,23 @@
+import graphene
+from django.core.exceptions import PermissionDenied
+from django.utils.translation import gettext as _
+
+from core.schema import OpenIMISMutation
+from payment_cycle.apps import PaymentCycleConfig
+from payment_cycle.services import PaymentCycleService
+
+
+class ProcessBenefitPlanPaymentCycleMutation:
+    _mutation_module = "payment_cycle"
+    _mutation_class = "ProcessPaymentCycleMutation"
```

Comment:
Can you either rename class to 'ProcessPaymentCycleMutation' or change the value of _mutation_class into 'ProcessBenefitPlan'?

Reply:
i fixed the _mutation_class